security-analysis/security-analysis-api/src/main/java/com/powsybl/security/json/OperatorStrategyDeserializer.java

```diff
@@ -73,7 +79,9 @@ public OperatorStrategy deserialize(JsonParser parser, DeserializationContext de
                     return false;
             }
         });
-        OperatorStrategy operatorStrategy = new OperatorStrategy(context.id, context.contingencyId, context.condition, context.actionIds);
+        ContingencyContext contingencyContext = new ContingencyContext(context.contingencyId,
+                context.contingencyContextType != null ? context.contingencyContextType : ContingencyContextType.SPECIFIC);
```

Comment:
maybe just add a note that this is done in this way to support both V1.0 and V1.1 JSON format - could be easily forgotten a couple of months from now ...

Reply:
Done